config/application.rb

```diff
@@ -25,5 +25,12 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
     # Load the integrations
     config.autoload_paths << Rails.root.join('lib', 'integrations')
+
+    config.middleware.insert_before 0, Rack::Cors do
```

Comment:
Wait, why? What do you need cors for?

You should probably only allow it for the specific places that you need it.